tempesta_fw/Makefile

```diff
@@ -42,9 +42,8 @@ tempesta_fw-objs = \
 	main.o \
 	pool.o \
 	sched.o \
-	server.o \
+	server_be.o \
```

Comment:
Yea, perhaps `server_backend` is better.
I just used `be` because I saw it several times across the project as an abbreviation for 'backend'.

I agree that `server_be` and `sock_frontend` are confusing.
Before these changes, there were separate `sock_backend` and `server` units.
`sock_backend` was responsible for connecting to back-end server, and `server` was responsible for `TfwServer` implementation.
And after these changes, I can't separate them in such way anymore because now there is a pool of connections, and basically almost all functions go to `sock_backend.c` and almost nothing is left for `server.c`.
I failed to make these units independed, so I merged them into the single unit.
Also, I can't name the new unit `sock_backend` because it is not only about connecting to back-end servers, but also about providing a specific `TfwServer` implementation. And I can't name it `server.c` because it is not abstract enough for the generic `TfwServer`, it is a kind of specific implementation of it.
I hope you understood my dilemma.